WearSpeakerSample/wear/src/main/java/com/example/android/wearable/speaker/ControlDashboard.kt

```diff
@@ -73,19 +74,19 @@ fun ControlDashboard(
         modifier = modifier
     ) {
         Spacer(
-            modifier = Modifier.layoutId(circle)
+            modifier = modifier.layoutId(circle)
         )
 
         ControlDashboardButton(
             buttonState = controlDashboardUiState.micState,
             onClick = onMicClicked,
             layoutId = mic,
-            imageVector = Icons.Filled.Mic,
+            imageVector = ImageVector.vectorResource(R.drawable.microphone),
```

Comment:
Why a local resource? Nothing in material icons?

Reply:
I removed the dependency to icons extended by error, it's now back with material icons.